bop_toolkit_lib/dataset/convert.py

```diff
@@ -0,0 +1,98 @@
+import argparse
```

Comment:
Maybe change the filename to `convert_v1_to_v2.py`? The versions could be more than just v1 and v2 later. Or we can add `--from`, `--to` in the arguments to support conversion for more versions.

Reply:
I renamed it to `convert_v1_to_v2.py`. We later have another more general script in `scripts/` to handle conversion between different formats.